Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "libc_alloc"
 description = "A simple global allocator which hooks into libc's malloc/free"
 authors = ["Daniel Prilik <danielprilik@gmail.com>"]
-version = "1.0.6"
+version = "1.0.7"
```

Comment:
No need to bump the version number - I prefer to have the version bump in a separate commit